lib/api/src/rest/schema.rs

```diff
@@ -64,6 +64,26 @@ pub enum ShardKeySelector {
     // ToDo: select by pattern
 }
 
+fn vector_example() -> Option<VectorStruct> {
+    Some(VectorStruct::Single(vec![
+        0.875, 0.140625, -0.15625, 0.96875,
+    ]))
+}
+
+fn payload_example() -> Option<segment::types::Payload> {
+    Some(Payload::from(serde_json::json!({
+        "city": "London",
+        "color": "green",
+    })))
+}
+
+fn shard_key_example() -> Option<segment::types::ShardKey> {
+    Some(ShardKey::from("shard-1"))
+}
+
+fn order_by_example() -> Option<segment::data_types::order_by::OrderValue> {
+    Some(segment::data_types::order_by::OrderValue::from(1))
+}
```

Comment:
why this is `1`?

Reply:
[order_value](https://github.com/qdrant/qdrant/blob/master/docs/grpc/docs.md#ordervalue) could be int64 or double.

`1 `is taken from [here](https://api.qdrant.tech/api-reference/search/recommend-points)